srcpkgs/freetube/template

```diff
@@ -0,0 +1,36 @@
+# Template file for 'freetube'
+pkgname=freetube
+version=0.7.0
+revision=1
+archs="x86_64"
+wrksrc=FreeTube-${version}-beta
```

Comment:
We only accept stable releases.

Reply:
Ok, no problem. Maybe I'll submit later when stable version arrives.
Thanks.